openshift/README.md

```diff
@@ -89,9 +89,43 @@ to remove the OpenShift project and all allocated AWS resources.
 1. In dev_console, `bayesian-data-importer` service is down.
 
 **cause**: In this case you have some data messed up in your Dynamo DB Tables.
-**resolution**: Completely remove **your tables only** from AWS Dynamo DB, i.e  tables prefixed with name `your_kerboros_*`. 
+**resolution**: Completely remove **your tables only** from AWS Dynamo DB, i.e  tables prefixed with name `your_kerberos_*`. 
 Redeploy `bayesian-gremlin-http` to recreate tables and then redeploy bayesian-data-importer.
 
+2. After dev_cluster delpoyment some pods failed with 'Timeout' issue.
+
+**cause**: This is happens as some of the keys in 'aws' and 'aws-dynamodb' secrets are missing.
+**resolution**: Log into dev cluster console and goto Your project --> Application --> Secret
+Manually add below keys in 'aws' secrets:
+    dynamodb-access-key-id: <ID>
+    dynamodb-secret-access-key: <KEY>
+    sqs-access-key-id: <ID>
+    sqs-secret-access-key: <KEY>
+
+Note: Replace <ID> and <KEY> with the id and key values present in other keys of same secret.
+
+Manually add below key in 'aws-dynamodb' secrets:
+    aws_region: dXMtZWFzdC0x
+
+Note: Value for this key is BASE64 encoded string of AWS region, in above case its 'us-east-1'
+
+3. Upon running ./deploy.sh or any script from terminal throws INVALID TOKEN error.
+
+**cause**: If dev-cluter session is timeout / user logs into again, then it generated a new login token for CLI.
+**resolution**: Always ensure that your OC_TOKEN value in env.sh is latest as per dev-cluster console value. Get the latest value from dev-cluster console.
+
+4. AWS console does not show my RDS / DynamoDB / DataPipe lines.
```

Comment:
`DataPipe lines.` has nothing to do with dev_cluster deployments. You may remove it

Reply:
Section #4, deleted from this readme